docs/commands.html

```diff
@@ -315,6 +315,21 @@ <h2><a name="hash">plz hash</a></h2>
     <p>The <code>--update</code> flag will cause Please to rewrite the BUILD file with
       any changed hashes that it can find.</p>
 
+  <h2><a name="fmt">plz fmt</a></h2>
+
+    <p>a.k.a. <code>plz format</code></p>
+
+    <p>Auto-formats existing BUILD files. You can either provide a list of files to reformat
+      or, if none are given, it will discover all BUILD files in the repository.</p>
+
+    <p>The <code>-w</code> flag rewrites existing files in-place; if not passed the
+      formatted version will be printed to stdout.</p>
+
+    <p>The implementation is currently based on a lightly modified version of
+      <a href="https://github.com/bazelbuild/buildtools">buildifier</a> which supports nearly
+      a superset of the same dialect, but lacks one or two features such as type annotations.<br/>
```

Comment:
Would be good to get annotations working too ;) 

We could then use this in build defs as well.